content/030_eksctl/launcheks.md

```diff
@@ -48,7 +48,7 @@ metadata:
   region: ${AWS_REGION}
   version: "1.19"
 
-availabilityZones: ["${AZS[0]}", "${AZS[1]}", "${AZS[2]}"]
+availabilityZones: ["${AZS[1]}", "${AZS[2]}", "${AZS[3]}"]
```

Comment:
Hi, thanks for this. The index of the car starts at 0, and I review it today, the actual code works fine. Can you verify on your side ?